### ISSUE TITLE
What patches should apply to windows 1511 version ? 

### ISSUE THREAD

**Issue by WestFarmer** [MicrosoftDocs/EdgeEnterprise#163]
I tried KB4520011, but it says not for my os.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: aae869a8-2ae4-d28f-6c85-c1190609f87b
* Version Independent ID: e2bcb1d6-27f7-5e30-ae6b-7acb5ee94098
* Content: [About IE mode](https://docs.microsoft.com/en-us/deployedge/edge-ie-mode)
* Content Source: [edgeenterprise/edge-ie-mode.md](https://github.com/MicrosoftDocs/EdgeEnterprise/blob/public/edgeenterprise/edge-ie-mode.md)
* Product: **microsoft-edge**
* GitHub Login: @dan-wesley
* Microsoft Alias: **kvice**

**Comment by dan-wesley** [comment]
@appcompatguy or @shisub can either one of you help with this?

**Comment by appcompatguy** [comment]
@dan-wesley - we don't support 1511 and haven't since October 10, 2017. We released these updates in 2019. 1511 also wasn't an LTSC build - 1507 and 1609 were. Since this build was out of service (and still is) I don't know if we ever made updates for it? @shisub do you know?

**Comment by dan-wesley** [comment]
@WestFarmer I reached out to a couple of our SMEs and this is the information they provided.

**Comment by WestFarmer** [comment]
if 1511 is not supported, can you mention it in documentation?

**Comment by appcompatguy** [comment]
We've listed all versions of Windows which are supported, so everything that isn't mentioned explicitly is unsupported.

We document the Windows support lifecycle here: https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet

1511 and 1703 fall into the same category - they weren't LTSC builds (which we do support) and were out of support when we shipped Microsoft Edge. A consequence of running an unsupported build of Windows is that you don't get security or functionality patches. We strongly recommend staying on a supported version of Windows.
